include/bout/constants.hxx

```diff
@@ -22,6 +22,10 @@ namespace SI {
   const BoutReal Me  = 9.10938356e-31;  ///< Electron mass
   const BoutReal Mp  = 1.672621898e-27; ///< Proton mass
   const BoutReal kb  = 1.38064852e-23;  ///< Boltzmanns constant
+  const BoutReal u   = 1.660539040e-27;        ///< Unified atomic mass unit
```

Comment:
Not sure about single character names. Perhaps `amu` or `uamu`?

Reply:
`u` and `Da` are the  official names: https://en.wikipedia.org/wiki/Unified_atomic_mass_unit
I haven't heard `Da` often, so I chose `u`.
I thought single character might be fine, as the name space is so small, but I am happy to change it ...

Reply:
My vote's for `amu`

Reply:
Agree; although according to that Wikipedia page `amu` is technically an obsolete unit, `amu` is used now to refer to `u` or `Da`. It probably doesn't matter, but my order of preference is `amu` > `Da` > `u`.

Reply:
Ok, changed :+1: